queries/c/locals.scm

```diff
@@ -26,9 +26,12 @@
   declarator: (field_identifier) @definition.field)
 (type_definition
   declarator: (type_identifier) @definition.type)
+(struct_specifier
+  name: (type_identifier) @definition.type)
 
 ;; References
 (identifier) @reference
+(type_identifier) @reference
```

Comment:
I don't think that type_identifiers are actually references.

Reply:
What is a reference for you? It references a type identifier when we can have type definitions we should also have type references. Or maybe add reference.type?

Reply:
I'll have a look again at which positions that node type can occur

Reply:
`type_reference` is the `Foo` in `Foo a = create_foo();` So references to type names.